internal/wrappers/container-resolver.go

```diff
@@ -1,20 +1,21 @@
 package wrappers
 
-import (
-	containersResolver "github.com/CheckmarxDev/containers-resolver/pkg/containerResolver"
-)
+import containersResolver "github.com/Checkmarx/containers-resolver/pkg/containerResolver"
 
 type ContainerResolverWrapper interface {
```

Comment:
what is this for? are u using it somewhere else that u need this class as an interface?

Reply:
Yes, it's interface in order for us to create mock container resolver for unit tests